product_docs/docs/pgd/5/manual/05-creating-cluster.mdx

```diff
@@ -0,0 +1,146 @@
+
+This initialises the first node.
+
+Now log into another initialised node's `bdrdb` database.
+
+Run `bdr.create_node` and give the node a different name and its connection string where *other* nodes may connect to it.
+
+Next, run `bdr.join_node_group` passing two parameters, the connection string for the first node and the name of the sub-group you want the node to join.
+
+Repeat these steps to initialise the third node.
+
+
+## Worked example
+
+So far, we have:
+
+* Created three Hosts.
+* Installed a Postgres server on each host.
+* Installed Postgres Distributed on each host.
+* Configured the Postgres server to work with PGD on each host.
+
+To create the cluster, we will tell `host-one`'s Postgres instance that it is a PGD node - `node-one` and create PGD groups on that node.
+Then we will tell `host-two` and `host-three`'s Postgres instances that they are PGD nodes - `node-two` and `node-three` and that they should join a group on `node-one`.
+
+### Create connection strings for each node
+
+We calculate the connection strings for each of the node in advance. 
+Below are the connection strings for our 3 node example:
+
+| Name       | Node Name  | Private IP      | Connection string                      |
+| ---------- | ---------- | --------------- | -------------------------------------- |
+| host-one   | node-one   | 192.168.254.166 | host=host-one dbname=bdrdb port=5444   |
+| host-two   | node-two   | 192.168.254.247 | host=host-two dbname=bdrdb port=5444   |
+| host-three | node-three | 192.167.254.135 | host=host-three dbname=bdrdb port=5444 |
+
+### Preparing the first node 
+
+Log into host-one's Postgres server.
+
+```
+ssh admin@host-one
+sudo -iu enterprisedb psql bdrdb
+```
+
+### Create the first node
+
+Call the [`bdr.create_node`](/pgd/latest/reference/nodes-management-interfaces#bdrcreate_node) function to create a node, passing it the node name and a connection string which other nodes can use to connect to it.
+
+```
+select bdr.create_node('node-one','host=host-one dbname=bdrdb port=5444');
+```
+
+#### Create the top-level group
+
+Call the [`bdr.create_node_group`](/pgd/latest/reference/nodes-management-interfaces#bdrcreate_node_group) function to create a top-level group for your PGD cluster. Passing a single string parameter will create the top-level group with that name. For our example, we will create a top-level group named `pgd`.
+
+```
+select bdr.create_node_group('pgd');
+```
+
+#### Create a sub-group
+
+Call the [`bdr.create_node_group`](/pgd/latest/reference/nodes-management-interfaces#bdrcreate_node_group) function again to create a sub-group of the top-level group. The sub-group name is the first parameter, the parent group is the second parameter. Using sub-groups to organise your nodes is recommended. For our example, we will create a sub-group `dc1` as a child of `pgd`.
+
```

Comment:
> Using sub-groups to organise your nodes is recommended

Where is this recommended?  The [main PGD docs](https://www.enterprisedb.com/docs/pgd/latest/node_management/groups_and_subgroups/) only say subgroups "can" be used. IMO it's a bit pointless to have a single subgroup for a basic 3-node test cluster (though subgroups would make a good item for "next steps" documentation).